### PR TITLE
MON-3518-filter-correction-recurrent-downtime-form

### PR DESCRIPTION
**Pull request by sc979**
Resolve MON-3518 on 2.8.x
As I couldn't push a coding style commit, on jben94's branch. 
I opened a new branch to add the required commit.

From jben94 PR -> 
Link issue : #6501
Link to the first PR : #6504 closed / rebased on the master
Link to the second PR PR : #6506

### CODE REVIEW

File: www/include/monitoring/recurrentDowntime/listDowntime.php [hunk 1]
```diff
@@ -46,6 +46,9 @@
 if (isset($_POST['searchDT']) && $_POST['searchDT']) {
     $search = $_POST['searchDT'];
     $downtime->setSearch($search);
+} elseif (isset($_GET['search']) && $_GET['search'] && !isset($_POST['searchDT'])) {
```
Comment (kduret) [inline www/include/monitoring/recurrentDowntime/listDowntime.php]: Why do you use search parameter ?
The good one is searchDT, so you can remove those lines.

**Comment by adr-mo** [comment]
@sc979 Could you please change the base for the PR from 2.8.x to master/18.10.x and rebase the branch.

**Comment by sc979** [comment]
no I can't. This PR is specific to the 2.8.x as we PEAR methods were replaced with PDO's
This one need to be merger on the 2.8.x. I'll make another one adapted to the master !

**Comment by adr-mo** [comment]
@sc979 I guess this one can be transformed only for master and 18.10.x branches.

No more bugfixes will be merged on the 2.8.x except critical/blocking bugs and security fixes

**Comment by sc979** [comment]
@adr-mo ok, I'll adapt it ASAP ;)

**Comment by lpinsivy** [comment]
on 2.8.x too please

**Comment by sc979** [comment]
the PR for the 18.10.x is here : 
https://github.com/centreon/centreon/pull/7201

**Comment by adr-mo** [comment]
**Validation ok for the 2.8.x it can now be merged.**

**Pull request merged**